src/Components/Web/src/Virtualization/IVirtualizeJsCallbacks.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
+
+namespace Microsoft.AspNetCore.Components.Virtualization
```

Comment:
Can this (and the actual components like Virtualize and InfiniteScroll) go into `Microsoft.AspNetCore.Components.Web`? This lines up with the package name and means people will have it in scope by default. This is intended as a mainstream scenario so we want people to find these awesome components easily :)